### DISCUSSION TITLE
How to install in micro sd?

### DISCUSSION THREAD

**Discussion by lucasmz1** [israpps/FreeMcBoot-Installer#37] [General]
how do I install in a micro sd in order to use it in mx4sio ? is it  possible? will it expand till my 8 gb size of my sd card or do I need it to use gparted or any other tool to resize my sd card? can I do it through my linux computer?

**Comment by israpps** [comment]
FreeMcBoot is not compatible in any form with MX4SIO.

And even if it was, you can't auto boot from an MX4SIO card because the PS2 can't do it on its own. It needs homebrew drivers to achieve it.

**Reply by lucasmz1** [reply]
Thanks for the explanation.

**Comment by lucasmz1** [comment]
however someone in reddit has tried that, I'll take a risk.
https://www.reddit.com/r/ps2homebrew/comments/16pcymj/guide_how_to_setup_fmcb_v1966_with_opl_for_mx4sio/

**Comment by israpps** [comment]
that tutorial is showing how to setup OPL to load games from MX4SIO. FreeMcBoot **can't** be installed into the MX4SIO

**Reply by lucasmz1** [reply]
you were right there's no way to use an mx4sio as an ordinary mencard. sad!

**Reply by israpps** [reply]
Take a look at SD2PSX, PSxMemCard Gen2 and memcardpro2 

They emulate memory card and can also be used as a better performant alternative to MX4SIO
